lib/wellies_web/handlers/response.ex

```diff
@@ -0,0 +1,17 @@
+defmodule WelliesWeb.ResponseHandler do
+
+  def get_body(response) do
+    case response do
+      {:ok, {{'HTTP/1.1', 200, 'OK'}, _headers, body}} ->
+        Jason.decode!(body)
+    end
+  end
+
+  def get_status_line(response) do
+    case response do
+      {_status, {status_line, _headers, _body}} ->
+        status_line
+    end
+  end
+
```

Comment:
Make sure you're consistent throughout the project with the leading spaces, etc.

If you're not sure if you have, try running `mix format` before committing. If you do `mix format` it will apply it to the whole project, so if you want it for a single file, do `mix format lib/sub_directory/my_file.ex`